feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -89,7 +91,7 @@ class IdentityManager(
     private fun generateRandomID(): String {
         return "\$RCAnonymousID:" + UUID.randomUUID().toString().toLowerCase(Locale.ROOT).replace("-", "")
             .also {
-                debugLog("Generated New App User ID - $it")
+                log(LogIntent.USER, IdentityStrings.NEW_ANON_ID)
```

Comment:
when should `USER` be used?

Reply:
whenever there's a change relating to App User IDs